config/k8saudit.js

```diff
@@ -0,0 +1,66 @@
+const { merge } = require('webpack-merge');
+var base = require('./index.js')
+
+module.exports = merge(base, {
+	app: {
+		host: 'www.audit.kiva.org',
+		publicPath: 'https://www-audit-kiva-org.freetls.fastly.net/ui/',
```

Comment:
Remove all other fields under `app`. Keep `publicPath` and remove `/ui/` from the end of it. Everything else can go as it's already defined in audit.js.